src/golang/lib/collections/operator/connector/load.go

```diff
@@ -72,3 +73,7 @@ func (l *Load) UnmarshalJSON(data []byte) error {
 	l.Parameters = params
 	return nil
 }
+
+func (l *Load) Scan(value interface{}) error {
+	return utils.ScanJSONB(value, l)
+}
```

Comment:
This was necessary to get the `json_extract(operator.spec, "$.load")` query working in sqlite/operators.go